vertex_launcher/src-tauri/src/download.rs

```diff
@@ -75,6 +78,27 @@ impl GameDownload {
             .emit(&self.event_name, self.get_state())
             .expect("Failed to broadcast the download progress");
     }
+    
+    pub async fn complete(&self){
+        let game_name = {
+            let game_list = LOCAL_GAME_LIST.read().await;
+            game_list
+                .get(&self.game_id)
+                .ok_or(GameListFetchError(format!(
+                    "Game with id {} not found",
+                    &self.game_id
+                ))).map_or(self.game_id.to_string(), |game| game.title.to_owned())
+                .to_owned()
+        };
+        
+        // by default show a notification
+        match notifica::notify("Vertex Launcher", &*format!("L'instalation de {} est terminée.", game_name)){
+            Ok(_) => {}
+            Err(e) => {
+                error!("Failed to get game name for notification body. {}", e);
+            }
+        }
```

Comment:
Use if let pattern if only one of your match case is used

```suggestion
        if let Err(e) = notifica::notify("Vertex Launcher", &*format!("L'instalation de {} est terminée.", game_name)){
            error!("Failed to get game name for notification body. {}", e);
        }
```

Reply:
In fact! Thanks for the review.